triggers/service/auth/src/main/scala/com/daml/auth/middleware/oauth2/Config.scala

```diff
@@ -79,6 +82,10 @@ object Config {
           "Maximum number of simultaneously pending login requests. Requests will be denied when exceeded until earlier requests have been completed or timed out."
         )
 
+      opt[Boolean]("cookie-secure")
+        .action((x, c) => c.copy(cookieSecure = x))
+        .text("Enable the Secure attribute on the cookie that stores the token.")
```

Comment:
maybe worth adding that this defaults to secure and that false should only be used for development?

Reply:
Good idea, done.